src/foam/u2/CurrencyView.js

```diff
@@ -19,7 +19,10 @@ foam.CLASS({
 
   methods: [
     function dataToText(val) {
-      return this.SUPER(val / 100);
+      val = Math.floor(val / 100);
```

Comment:
FloatView has a formatNumber method on it that is used by its dataToText method. You might want to override that here and leave this method as is.